yarn.lock

```diff
@@ -1368,6 +1368,14 @@
   dependencies:
     "@types/react" "*"
 
+"@types/react-native@*":
```

Comment:
Your lock file should have no changes in it since there should be no dependency changes on this PR.